src/gantt.ts

```diff
@@ -1209,15 +1312,8 @@ export class Gantt implements IVisual {
 
     public static sortTasksWithParents(tasks: Task[], sortingOptions: SortingOptions): Task[] {
         const sortingFunction = ((a: Task, b: Task) => {
```

Comment:
There is a JS string method `localeCompare`, what do you think about using it here? It will work a little bit slower, but works with user locale, what might be useful for some users. Please take a look on this implementation and let me know your opinion (also it looks more laconic).
```
const sortingFunction = (a: Task, b: Task) => {
            const direction = sortingOptions.sortingDirection === SortDirection.Ascending ? 1 : -1;
            return a.name.localeCompare(b.name, undefined, {numeric: true}) * direction;
        };
```